### PR TITLE
feat: update utxo & update edition on transfer modal

### PR DESCRIPTION
**Pull request by itttm127**
- xcpService.ts
Updated the condition on 589 ~ 610
- stampRepository.ts
Updated the unbound_quantity count on 813 ~ 818
- WalletTransferModal.tsx
Added the function to get max quantity from stamps on  126 ~ 136
Added the handleQuantityChange function when quantity input change on  112 ~ 125

### CODE REVIEW

File: server/database/stampRepository.ts [hunk 1]
```diff
@@ -762,24 +794,34 @@ export class StampRepository {
...
+          .reduce((acc, balance) => acc + balance.quantity, 0);
+        const utxos = xcpBalances
+          .filter(
+            (balance) => balance.cpid === summary.cpid && balance.utxo_address
+          )
+          .map((balance) => ({
+            quantity: balance.quantity,
+            utxo_address: balance.utxo_address,
+          }));
+        const unbound_quantity = xcpBalances
```
Comment (reinamora137) [inline server/database/stampRepository.ts:813]: this is the database call which shouldn't differentiate between bound and unbound here. we need to typically keep all balances the total balance with the exception of the /api/v2/stamps/balance endpoing

Comment (itttm127) [reply server/database/stampRepository.ts:813]: I think that's not database call 
it's just using the javascript for filtering data.

**Comment by reinamora137** [comment]
Issue #435

File: server/database/stampRepository.ts (hunk 1 above)
Comment (reinamora137) [reply server/database/stampRepository.ts:813]: ah ok i see. this looked like it was part of getstamps

Comment (itttm127) [reply server/database/stampRepository.ts:813]: 👍

**Pull request merged**